### ISSUE TITLE
BE Filters

### ISSUE THREAD

**Issue by rjclarkewp007** [horbin/FuMS-HC-Exile#16]
Hi

Any chance the BE filters can be updated?
I had to disable BE for now as it keeps kicking players when running this mod.

thanks!

**Comment by rjclarkewp007** [comment]
I receive this error publicvariable.log:

#0 "FuMS_GetPlayerIndex" = R Alpha 1-1:1

I tried to add this to publicvairable.txt
7 "" !="FuMS_GetPlayerIndex"

still same error. :(

**Comment by rjclarkewp007** [comment]
> 
> 
> I receive this error publicvariable.log:
> 
> #0 "FuMS_GetPlayerIndex" = R Alpha 1-1:1
> 
> I tried to add this to publicvairable.txt
> 7 "" !="FuMS_GetPlayerIndex"
> 
> still same error. :(

**Comment by ExiledHeisenberg** [comment]
No, I don't use this system, so if you do it can you share the data please?  I'll update the filters here
